en/micro-integrator/docs/references/connectors/configure-connector-exporter-project.md

```diff
@@ -0,0 +1,27 @@
+## Configure the connector in WSO2 Integration Studio
+
+1. Open WSO2 Integration Studio and create an ESB Solution Project.
+<p><img src="../../assets/img/connectors/solution-project.png" title="Creating a new ESB Solution Project" width="800" alt="Creating a new ESB Solution Project" /></p>
+
+2. Right click on the project that you created and click on 'Add or Remove Connector' -> Add Connector.
+<p><img src="../../assets/img/connectors/add-connector.png" title="Add or Remove Connector" width="800" alt="Add or Remove Connector" /></p>
+
+3. It will be navigated to the WSO2 Connector Store. 
+4. Search for File Connector and download it to the workspace. 
+<p><img src="../../assets/img/connectors/search-connector.png" title="Search Connector in the Connector Store" width="800" alt="Search Connector in the Connector Store" /></p>
+
+5. Now it needs to add a Connector Exporter Project. 
```

Comment:
```suggestion
5. Now you need to add a Connector Exporter Project. 
```

Merge this point with the next one. All items in the list must have actions.

Reply:
This line was removed.